package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "myl3",
+  "description": "My Lifelong Learning Lab",
+  "version": "0.1.0-SNAPSHOT",
+  "author": "Fluid Project",
+  "bugs": "https://issues.fluidproject.org/issues/?jql=project+%3D+FLOE+AND+component+%3D+%22Learning+Dashboard%22",
+  "homepage": "http://www.fluidproject.org/",
+  "dependencies": {
+      "pouchdb":"5.4.3"
```

Comment:
Would it make sense to pull in Infusion as a dependency here as well, or is the npm module missing something you need?